src/Evolution/Systems/Cce/InterfaceManagers/GhLocalTimeStepping.hpp

```diff
@@ -110,7 +110,7 @@ class GhLocalTimeStepping : public GhInterfaceManager {
   void clean_up_gh_data() noexcept;
 
   std::map<double, gh_variables> gh_data_;
-  std::deque<TimeStepId> requests_;
+  std::set<TimeStepId> requests_;
```

Comment:
I think `std::priority_queue` is that data structure you're looking for, unless you need to ignore duplicates for some reason.

Reply:
Ah, the `std::priority_queue` turns out to be troublesome for serialization, because as far as I can tell, it's a non-iterable container. The only way I can see to serialize it is to either make a full copy and serialize as elements are popped off the top, or to pop elements, save them off to a different container, then push them back on again (so, again, requires something like a copy).

If it's acceptable for the time being, I think I'd prefer to leave it as a set to make the serialization more graceful.

Reply:
Ah, yeah, that's a good reason.  I expect this container should always be small anyway.